qiita_db/test/test_study.py

```diff
@@ -147,11 +145,11 @@ def setUp(self):
             'metadata_complete': True,
             'reprocess': False,
             'number_samples_promised': 27,
-            'emp_person_id': qdb.study.StudyPerson(2),
+            'emp_person_id': 2,
```

Comment:
@antgonza @ElDeveloper @mortonjt `emp_person` is not used anywhere in the code and I think we should drop it as that is something that we will be keeping track in Jira, not Qiita. Should we spend time fixing that or just agree that we are going to drop it?